openframe-external-api-service-core/src/main/java/com/openframe/external/mapper/.DeviceMapper.md

```
@@ -33,4 +33,6 @@ DeviceFilters filters = filterService.getAvailableFilters();
 DeviceFilterResponse filterResponse = deviceMapper.toDeviceFilterResponse(filters);
 ```
 
+> `toDeviceResponse` passes through the device's `hostname`, `displayName`, and `nickname` from the `Machine` document.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)